.github/workflows/ml.yaml

```diff
@@ -87,11 +87,12 @@ jobs:
       run: |
         # run each package test suite; append to coverage file
         # Exit immediately if a command exits with a non-zero status.
-        set -e
+        # set -e
```

Comment:
should this be uncommented?

Reply:
woops, forgot to undo that when I was poking around